api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -900,4 +897,8 @@ private String userForLogging(DbUser user) {
   private String ownersForLogging(Collection<DbUser> owners) {
     return owners.stream().map(this::userForLogging).collect(Collectors.joining(", "));
   }
+
+  private List<String> ownersEmailList(Collection<DbUser> owners) {
```

Comment:
I don't like plain "email" here because that can also mean username.  How about `ownerContactEmails()` ?